abstract_specification.md

```diff
@@ -465,26 +763,24 @@ Parameters:
 
 Summary of operations:
 
-| Operation                  | Description                                                                    |
-| -------------------------- | ------------------------------------------------------------------------------ |
-| create(uri, ...)           | Create a SOMASparseNDArray named with the URI.                                 |
-| delete(uri)                | Delete the SOMASparseNDArray specified with the URI.                           |
-| exists(uri) -> bool        | Return true if object exists and is a SOMASparseNDArray.                       |
-| get soma_type              | Returns the constant "SOMASparseNDArray"                                       |
-| get shape -> (int, ...)    | Return length of each dimension, always a list of length `ndim`.               |
-| get ndim ->  int           | Return number of dimensions.                                                   |
-| get schema -> Arrow.Schema | Return data schema, in the form of an Arrow Schema                             |
-| get is_sparse -> True      | Return the constant True.                                                      |
-| get nnz -> uint            | Return the number stored values in the array, including explicit zeros.        |
-| read                       | Read a slice of data from the SOMASparseNDArray.                               |
-| write                      | Write a slice of data to the SOMASparseNDArray.                                |
+| Operation                                    | Description                                                             |
+| -------------------------------------------- | ----------------------------------------------------------------------- |
+| static create(uri, ...) -> SOMASparseNDArray | Create a SOMASparseNDArray named with the URI.                          |
+| get soma_type                                | Returns the constant "SOMASparseNDArray".                               |
+| get shape -> (int, ...)                      | Return length of each dimension, always a list of length `ndim`.        |
+| get ndim -> int                              | Return number of dimensions.                                            |
+| get schema -> Arrow.Schema                   | Return data schema, in the form of an Arrow Schema.                     |
+| get is_sparse -> True                        | Return the constant True.                                               |
+| get nnz -> uint                              | Return the number stored values in the array, including explicit zeros. |
+| read                                         | Read a slice of data from the SOMASparseNDArray.                        |
+| write                                        | Write a slice of data to the SOMASparseNDArray.                         |
```

Comment:
important to use `slice` here instead of `subset` as we do above? If it's a technical term we explicitly want, ok.